app/views/admin/access_overviews/index.haml

```diff
@@ -0,0 +1,92 @@
+- content_for :title, 'Access Overview'
```

Comment:
It feels like HMIS and Warehouse should share this content. What do you think?>

Reply:
There are some small discrepancies in the language and links, and I'm not convinced they'll continue to be so similar, so for now the duplication (or almost duplication) seems reasonable.